src/SFA.DAS.EmployerIncentives.Commands/Services/BusinessCentralApi/PaymentsSuccessfullySent.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections.Generic;
+using SFA.DAS.EmployerIncentives.Abstractions.DTOs.Queries.ApprenticeshipIncentives;
+
+namespace SFA.DAS.EmployerIncentives.Commands.Services.BusinessCentralApi
+{
+    public class PaymentsSuccessfullySent
```

Comment:
Can this be renamed as SendPaymentsResponse since the data contains what was successfully sent?

Reply:
done